### ISSUE TITLE
[Bug]: The API endpoint to create a single status is missing the alt-text field.

### ISSUE THREAD

**Issue by TinoJ** [pixelfed/pixelfed#5937]
### Description

Hello,

It looks like the alt text is missing from the API.
https://beta-preview.pixelfed.io/resources/statuses#create-a-single-media-status
It's also not listed in the Create Status and Edit Status options.

I could not find a bug report on that issue.

Or am I missing something?

Best, Tino

### Steps to reproduce

I only rely on the documentation.
I am on pixelfed.social.

### Pixelfed version

v0.12.5

### Acknowledgements

Yes

**Comment by eufelipemateus** [comment]
I didn’t even know this documentation existed lol, and I’ve had this instance since 2022!
